src/views/network/view_session.cpp

```diff
@@ -97,7 +97,9 @@ namespace big
 
 		components::sub_title("Chat");
 		ImGui::Checkbox("Auto-kick Chat Spammers", &g.session.kick_chat_spammers);
-		ImGui::Checkbox("Disable Filter", &g.session.disable_chat_filter);
+		ImGui::Checkbox("Force Clean", &g.session.chat_force_clean);
```

Comment:
Force Clean Disabled*

Reply:
man please stop about grammar literally nobody cares,.
it's going to be generalized soon anyways, translations are coming.

Reply:
> and please stop commenting about grammar literally nobody cares, it's going to be generalized soon anyways, translations are coming.

im just tryna point out it would confuse people. Sure it will be generalized soon but that doesnt mean that quality should be frowned upon until the translation service comes out

Reply:
bro there is a tooltip directly below for retards if you simply cannot understand context or the english language

Reply:
> bro there is a tooltip directly below for retards if they simply cannot understand context or the english language

I'm going to be honest, I agree with @TheGreenBandit here. I didn't even know what the point of the feature was til I looked at the code. The name isn't very suggestive. "Disable Filter" is more explanatory than "Force Clean".

But that's just my take, do what you want. I'm not a narcissist.